app/controllers/games_controller.rb

```diff
@@ -23,6 +23,18 @@ def show
     @game = Game.find(params[:id])
   end
 
+  def forfeit_game
+    @game = Game.find(params[:id])
+
+    if @game.user_id_black != current_user
```

Comment:
Does this work as expected?  Since `user_id_black` is an integer, you might need to compare it to `current_user.id`.